components/expenses/ExpenseCategorySelect.tsx

```diff
@@ -133,12 +139,11 @@ const getCategoryLabel = (
 /**
  * Returns true if the category is supported for the given expense type. Host admins can select any category.
  */
-export const isSupportedExpenseCategory = (
-  expenseType: ExpenseType,
-  category: AccountingCategory,
-  isHostAdmin: boolean,
-) => {
-  return isHostAdmin || !category?.expensesTypes || category.expensesTypes.includes(expenseType);
+export const isSupportedExpenseCategory = (expenseType: ExpenseType, category: AccountingCategory) => {
+  return (
+    category?.kind === AccountingCategoryKind.EXPENSE &&
+    (!category?.expensesTypes || category.expensesTypes.includes(expenseType))
+  );
```

Comment:
The part with `isHostAdmin` was not a mistake, as per the description: "Host admins can select any category.". It's a strange requirement that OCF had. I would propose:

```suggestion
export const isSupportedExpenseCategory = (expenseType: ExpenseType, category: AccountingCategory, isHostAdmin: boolean) => {
  return (
    category?.kind === AccountingCategoryKind.EXPENSE &&
    (isHostAdmin || !category?.expensesTypes || category.expensesTypes.includes(expenseType))
  );
```